internal/services/network/public_ips_data_source_test.go

```diff
@@ -28,40 +28,6 @@ func TestAccPublicIPsDataSource_namePrefix(t *testing.T) {
 	})
 }
 
-func TestAccPublicIPsDataSource_assigned(t *testing.T) {
```

Comment:
Why is this removed? not supported?

Reply:
Because in the config it tries to reference to an unavailable resource, the name of resource is `azurestack_nat_gateway`